cfgov/paying_for_college/disclosures/scripts/update_colleges.py

```diff
@@ -72,92 +78,119 @@ def set_school_grad_rate(school, api_data):
-            school = set_school_grad_rate(school, data)
-            program_data = api_utils.compile_school_programs(data)
-            if program_data and type(program_data.get('most_popular')) == list:
-                UPDATED = True
-                school.program_most_popular = program_data['most_popular']
-                school.program_count = program_data.get(
-                    'program_count')
-            if UPDATED is True:
-                UPDATE_COUNT += 1
-                school.zip5 = fix_zip5(str(school.zip5))
-                school.save()
+        sys.stdout.write('.')
+        sys.stdout.flush()
+        update_count += 1
+        if data.get('school.operating') == 0:
+            school.operating = False
+            school.save()
+            closed.append(school)
+            continue
+        data['school.operating'] = True
+        for key in DECIMAL_MAP:
+            if data.get(key) is not None:
+                setattr(school, DECIMAL_MAP[key], Decimal(str(data[key])))
+        for key in MODEL_MAP:
+            if data.get(key) is not None:
+                setattr(school, MODEL_MAP[key], data[key])
+        if data.get('school.ownership'):
+            school.ownership = str(data['school.ownership'])
+            school.control = CONTROL_MAP[school.ownership]
+        set_school_grad_rate(school, data)
+        compile_net_prices(school, data)
+        program_data = api_utils.compile_school_programs(data)
+        if program_data and type(program_data.get('most_popular')) == list:
+            school.program_most_popular = program_data['most_popular']
+            school.program_count = program_data.get(
+                'program_count')
+        school.zip5 = fix_zip5(str(school.zip5))
+        school.save()
     endmsg = """
     Updated {} schools and found no data for {}\n\
     Schools that closed since last run: {}\n\
     \n{} took {} to run""".format(
-        UPDATE_COUNT,
-        len(NO_DATA),
-        len(CLOSED),
+        update_count,
+        len(no_data),
+        len(closed),
         SCRIPTNAME,
-        (datetime.datetime.now() - STARTER))
-    if NO_DATA:
-        logger.info("\n\nSchools for which we found no data:")
-        for school in NO_DATA:
-            logger.info("- {} ({})".format(school.primary_alias, school.pk))
-    if CLOSED:
+        (datetime.datetime.now() - starter))
+    if no_data:
+        logger.info("\n\nSchools for which we found no data on {}:".format(
+            datetime.date.today().strftime("%b %-d, %Y")))
+        for school in no_data:
+            logger.info("- {}".format(school))
+    if closed:
         logger.info("\n\nSchools that have closed since the last update:")
-        for school in CLOSED:
-            logger.info("- {} ({})".format(school.primary_alias, school.pk))
-
-    return (NO_DATA, endmsg)
+        for school in closed:
+            logger.info("- {}".format(school))
+    logger.info(endmsg)
```

Comment:
The addition of this line means that the output get written twice, since it's already being written out by the `update_via_api` command [here](https://github.com/cfpb/cfgov-refresh/blob/01a7865e944bc7f1363a818cf58114b598b1f64e/cfgov/paying_for_college/management/commands/update_via_api.py#L27).

```suggestion
```